### PR TITLE
replace deprecated egrep with grep -e to suppress warning

### PR DESCRIPTION
**Pull request by coleyoungsh**
egrep is deprecated

### CODE REVIEW

File: vv [hunk 1]
```diff
@@ -540,7 +540,7 @@ inittermkey() {
 
     # To be valid, we must enable terminal application mode key sequences.
     if tput smkx; then
-	for k in $(infocmp -L1 | egrep 'key_|cursor_'); do
+	for k in $(infocmp -L1 | grep -e 'key_|cursor_'); do
```
Comment (hackerb9) [inline vv:543]: Thank you, but  'grep -e' is not equivalent to 'egrep'. POSIX wants everyone to type `grep -E`, I believe.

**Comment by hackerb9** [comment]
Thank you for the report. I believe I have fixed it in the latest version.

**Comment by coleyoungsh** [comment]
Oh my mistake, just trying to quick fix. I’ll update when I can ✌️

**Comment by hackerb9** [comment]
No need to update. I've fixed it. Thanks again for the report.

**Pull request closed without merging**